internal/cmd/cmd_decr.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"math"
+
+	"github.com/dicedb/dice/internal/object"
+	dstore "github.com/dicedb/dice/internal/store"
+	"github.com/dicedb/dice/wire"
+)
+
+var cDECR = &DiceDBCommand{
+	Name:      "DECR",
+	HelpShort: "DECR decrements the value of the specified key in args by 1",
+	Eval:      evalDECR,
+}
+
+func init() {
+	commandRegistry.AddCommand(cDECR)
+}
+
+//nolint:gocyclo
```

Comment:
Are there enough cyclomatic complexity that you need to add this flag? I don't feel so. Check once.